### ISSUE TITLE
Recipients dropdown field not validating

### ISSUE THREAD

**Issue by TomDeSmet** [verbb/formie#574]
**Description**
I have a recipients dropdown field that is set to required but is not validating on the front-end.
I think the issue may be that the required attribute is not filled in correctly.
If I change the required attributes `true` value to be a String in the template code, strangely it is filled in but still I don't get the default error message.

I am overwriting these templates, but I have made them up-to-date and I also tried by removing my custom templates and using the default ones, without avail.

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.4.16
- Craft version: 3.7.12
- Multi-site: Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.16",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.17](https://github.com/verbb/formie/releases/tag/1.4.17)
